src/frontend/apps/desk/src/features/teams/api/useTeams.tsx

```diff
@@ -24,15 +50,16 @@ export const getTeams = async () => {
 export const KEY_LIST_TEAM = 'teams';
 
 export function useTeams(
+  param: TeamsParams,
   queryConfig?: UseQueryOptions<
     TeamsResponse,
     TeamsResponseError,
     TeamsResponse
   >,
 ) {
   return useQuery<TeamsResponse, TeamsResponseError, TeamsResponse>({
```

Comment:
Should we use [useInfiniteQuery](https://tanstack.com/query/latest/docs/framework/react/reference/useInfiniteQuery) instead?

Reply:
👍  It gives us a good refacto
https://github.com/numerique-gouv/people/pull/31/commits/ccef7d82b26ff1a60c90314be58c89d48aa38580

Reply:
alright, good !